core/src/test/java/io/grpc/internal/ServiceConfigInterceptorTest.java

```diff
@@ -58,7 +59,8 @@ public void setUp() {
     MockitoAnnotations.initMocks(this);
   }
 
-  private final ServiceConfigInterceptor interceptor = new ServiceConfigInterceptor();
+  private final ServiceConfigInterceptor interceptor = new ServiceConfigInterceptor(
+      true /* retryEnabled */, 5 /* maxRetryAttemptsLimit */);
```

Comment:
nit: these comments need to have a `=` after the variable name, or the linter will complain